examples/blockly-parcel/package.json

```diff
@@ -9,12 +9,12 @@
     "build": "parcel build public/index.html"
   },
   "devDependencies": {
-    "blockly": "^5.2",
+    "blockly": "^8.0.5",
```

Comment:
Please make sure to update `package-lock.json` to match (`npm install` should do the trick).